packages/astro/src/content/consts.ts

```diff
@@ -34,6 +34,10 @@ export const CONTENT_FLAGS = [
 ] as const;
 
 export const CONTENT_TYPES_FILE = 'content.d.ts';
+
+export const DATA_STORE_MANIFEST_FILE = 'data-store/__manifest.json';
```

Comment:
I would not include the directory in here. In any case, it's a bad idea to include path separators in constants like this, because they're platform-dependent.